UnityProject/Assets/Scripts/Player/PlayerSprites.cs

```diff
@@ -382,6 +383,8 @@ public void SetSurfaceColour()
 			ColorUtility.TryParseHtmlString(ThisCharacter.SkinTone, out CurrentSurfaceColour);
 		}
 
+		CurrentSurfaceColour.a = 1;
```

Comment:
can you fix the bug with using == / contains with colour when it needs to be a approximate check because float numbers, needs to be on the customisations sprites as well